subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/junitplatform/JUnitPlatformOptions.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.tasks.testing.junitplatform;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.tasks.testing.TestFrameworkOptions;
+
+/**
+ * The JUnit platform specific test options. To be implemented in next step.
+ * @since 4.6
+ */
+@Incubating
+public class JUnitPlatformOptions extends TestFrameworkOptions{
```

Comment:
I guess that Checkstyle will complain about the missing space between `TestFrameworkOptions` and `{`.

Reply:
Thanks, done.

Reply:
done